### ISSUE TITLE
Backup and download Raw JSON does not contain uploaded PDF

### ISSUE THREAD

**Issue by haberjr** [remnoteio/remnote-issues#506]
I have uploaded a pdf file.
End of kiddy I export raw JSON
Then import in desktop Mac  remnote version
PDF is reported as missing rem

**Comment by eohomegrownapps** [comment]
This problem also seems to hold for uploaded images.

**Comment by haberjr** [comment]
Yes, I have also notice that sometimes an image uploaded is not part of the Raw files when you export and consequently does not show up in the offline version after import.

**Comment by orkhan10** [comment]
It says "missing rem" however it opened in dual pane view.  However was not able to right click on it. And as soon as I uploaded another pdf into the same document the "missing rem" (the first document) disappered.

**Comment by orkhan10** [comment]
And the aliases to pdf references also disappear. The original pdf text is shown not the edited alias.
